src/modalities/utils/gradient_clipping.py

```diff
@@ -0,0 +1,18 @@
+from typing import Callable
+
+from torch.nn.utils import clip_grad_norm_, clip_grad_value_
+
+from modalities.config.config import GradientClippingMode
+from modalities.models.model import NNModel
+
+
+def build_gradient_clipper(
+    gradient_clipping_mode: GradientClippingMode, gradient_clipping_threshold: float
+) -> Callable[[NNModel], None]:
+    if gradient_clipping_mode == GradientClippingMode.P2_NORM:
+        return lambda model: (clip_grad_norm_(model.parameters(), gradient_clipping_threshold, 2), None)[-1]
```

Comment:
what is the `None` for? Also, wouldn't the combination with `[-1]` always return `None`?